doc/dev/installation.rst

```diff
@@ -63,13 +63,80 @@ from within the root of the running container.
   first, open it as a VS Code Workspace, and then reopen the Workspace in a Dev Container via the
   ``Reopen in Container`` command.
 
-Building from source
---------------------
+
+
+Minimal Building From Source Guide
+----------------------------------
+
+
+Most developers might want to build Catalyst from source instead of using a pre-shipped package. In this section we present a minimal building-from-source installation guide. The next section provides a more detailed guide, which we **strongly** recommend the user to read through. Importantly, each component of Catalyst, namely the Python frontend, the MLIR compiler, and the runtime library, can be built and tested indenpendently, which this minimal installation guide does not go over. 
```

Comment:
```suggestion
Most developers might want to build Catalyst from source instead of using a pre-shipped package. In this section we present a minimal building-from-source installation guide.

The next section provides a more detailed guide, which we **strongly** recommend the user to read through. Importantly, each component of Catalyst, namely the Python frontend, the MLIR compiler, and the runtime library, can be built and tested indenpendently, which this minimal installation guide does not go over. 
```